Dockerfile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_CONTAINER=nvcr.io/nvidia/pytorch:23.10-py3
+ARG BASE_CONTAINER=nvcr.io/nvidia/pytorch:23.12-py3
```

Comment:
Wasnt this updated for the release?

Reply:
For this release, the base image was changed manually in the internal build pipelines using the `BASE_CONTAINER` arg.